Integrations/integration-Lastline.yml

```diff
@@ -147,10 +144,16 @@ script:
                 case 106:
                     // analysis report not ready
                     break;
+                case 115:
+                    // reached api quota
+                    throw "You've reached your API call quota. Contact your Lastline representative.";
```

Comment:
instead of throwing please change it to return error entry, or catch the error at the higher level and return error entry. When we are throwing it shows ugly stack trace

Reply:
true. done.